Makefile

```diff
@@ -497,4 +498,10 @@ deploy-from-index-image:
 .PHONY: test-acceptance-with-bundle
 ## Run acceptance tests with the operator installed from a given index image and channel
 test-acceptance-with-bundle: deploy-from-index-image
-	$(Q)TEST_ACCEPTANCE_START_SBO=remote $(MAKE) test-acceptance
\ No newline at end of file
+	$(Q)TEST_ACCEPTANCE_START_SBO=remote $(MAKE) test-acceptance
+
+.PHONY: release-manifests
+## prepare file for Release
+release-manifests: setup-venv release-operator prepare-operatorhub-pr get-test-namespace
+	$(Q)$(PYTHON_VENV_DIR)/bin/pip install pyaml
```

Comment:
this should happen as a part of `setup-env`. Add `pyaml` to `requirements.txt`

Reply:
This requirements.txt https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/requirements.txt ? Btw this requirements.txt is setup during test-acceptance-setup target, but I just want to  use setup-venv. I am using setup-venv target since the release-manifests target is not solely for acceptance tests